src/core/thread/mle_router.cpp

```diff
@@ -451,7 +440,7 @@ void MleRouter::SetStateLeader(uint16_t aRloc16, LeaderStartMode aStartMode)
     Get<Mac::Mac>().UpdateCsl();
 #endif
 
-    LogNote("Leader partition id 0x%lx", ToUlong(mLeaderData.GetPartitionId()));
+    LogNote("Leader partition ID 0x%lx", ToUlong(mLeaderData.GetPartitionId()));
```

Comment:
nit: since this is generic to both leader and router?
```suggestion
    LogNote("Partition ID 0x%lx", ToUlong(mLeaderData.GetPartitionId()));
```